aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -37,7 +37,7 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
 
 ## Prerequisites
 
-* [.NET Core 2.2 SDK](https://www.microsoft.com/net/core)
+* [.NET Core SDK](https://www.microsoft.com/net/core)
```

Comment:
@Rick-Anderson The topic isn't versioned in metadata, but does it imply "2.2 or later"? If so, it should go ... and I think that's the wrong link. That link resolves to this :point_right: https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/intro

```suggestion
* [.NET Core SDK 2.2 or later](https://dotnet.microsoft.com/download)
```

Reply:
I changed the target URL as suggested, that makes sense.